Providers/Modules/Plugins/Common/plugin/out_oms_blob.rb

```diff
@@ -385,7 +385,6 @@ def handle_records(tag, records)
     # This method is called when an event reaches to Fluentd.
     # Convert the event to a raw string.
     def format(tag, time, record)
-      @log.trace "Buffering #{tag}"
```

Comment:
@abenbachir why remove ?

Reply:
This function format(tag, time, record) is part of hotpath, it will be called for each event going to that output pluging, we should make sure to put minimum stuff there.

This line @log.trace "Buffering #{tag}" will create a string each time even if trace loggging is disabled. "Buffering #{tag}" is always more than 23 length, so ruby will call malloc to store it.